src/stores/MapOptionsStore.ts

```diff
@@ -208,7 +213,7 @@ export default class MapOptionsStore extends Store<MapOptionsStoreState> {
                 `Could not find tile layer specified in config: '${config.defaultTiles}', using default instead`
             )
         return {
-            selectedStyle: selectedStyle ? selectedStyle : mapTiler,
+            selectedStyle: selectedStyle ? selectedStyle : osmOrg,
```

Comment:
We probably want to change this to Omniscale

Reply:
And shall we even keep map tiler?

Reply:
Let's use Omniscale as default (but keep map tiler)

Reply:
But as maptiler and mapilion are so slow we should probably mark them as experimental (e.g. gray out the label a bit?)

Reply:
Ok I made Omniscale the default now. If the two vector layers are so slow that they should not be used why not remove them altogether? Or is there any case where they would be useful?

Reply:
For testing and improving I would keep them. We could then ask / contract e.g. OL maintainers to improve this somehow :)